package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,90 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=True, superposition=False):
```

Comment:
I noted it in the function doc-string and the Changelog. I also planned to add this to the blog post as it is a more major API break.